pkg/api/v1/schema.go

```diff
@@ -920,4 +920,9 @@ type CniNetwork struct {
 	// specified, VMI namespace is assumed.
 	// In case of genie, it references the CNI plugin name.
 	NetworkName string `json:"networkName"`
+
+	// For Multus CNI select the default network and add it to the
+	// multus-cni.io/default-network annotation. Ignored for all
```

Comment:
it's not ignored, it is actually forbidden by the validating hook

Reply:
Originally put here before I added that logic. I'll update the text.

Reply:
Fixed.